The output model not have to inherit! (#13611)
dotnet/docs: docs/machine-learning/how-to-guides/machine-learning-model-predictions-ml-net.md

}
```

> Here, the `HousingPrediction` data model inherits from `HousingData` to make it easy to visualize the original input data along with the output generated by the model. But the data model does not have to inherit from the input model, especially in cases where the input data is being transformed during the training, so the output model no longer represents the data types.  

Reviewer: ```suggestion
In this case, the `HousingPrediction` data model inherits from `HousingData` to make it easy to visualize the original input data along with the output generated by the model. Note that the output data model does not have to inherit from the input data model, especially in cases where the input data is transformed during training, since the output data no longer represents the transformed data.  
```

Reviewer: Thanks for this PR @baruchiro. I've added some clarifications.

Reviewer 2: I wonder whether we remove the inheritance for this section of the article. It is a convenience that could (and was) misleading to some users.

Reviewer: That's an even better option. @baruchiro would you be able to make that change? Essentially replace the existing `HousingPrediction` class with the following code.

```
class HousingPrediction
{
    [ColumnName("Score")]
    public float PredictedPrice { get; set; }
}
```

Then, also removing the paragraph you made updates to explaining the inheritance since it is no longer applicable.
